inference-engine/src/multi_device/CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ ie_add_plugin(NAME ${TARGET_NAME}
               SOURCES ${SOURCES} ${HEADERS}
               VERSION_DEFINES_FOR multi_device_plugin.cpp)
 
+ie_add_plugin(NAME ${TARGET_NAME}
+              DEVICE_NAME "AUTO"
+              PSEUDO_PLUGIN
+              DEFAULT_CONFIG "MULTI_WORK_MODE_AS_AUTO:")
```

Comment:
btw, what the ":" is for in the default config?

Reply:
is it becuase of "key:val" structure?

Reply:
Used more explicit syntax:
```cmake

ie_add_plugin(NAME ${TARGET_NAME}
              DEVICE_NAME "AUTO"
              PSEUDO_PLUGIN
              DEFAULT_CONFIG "MULTI_WORK_MODE_AS_AUTO:YES")
```